content/docs/releases/v1.1.0.md

```diff
@@ -0,0 +1,84 @@
+- [NSM Bug](https://github.com/networkservicemesh/integration-k8s-gke/issues/215)
+- Workaround: [Downgrade to GKE Node version 1.20.10-gke.301](https://github.com/networkservicemesh/integration-k8s-gke/commit/c181a994e334582237ff1d01e40218dab076a3e0#diff-7fbc346a181ddc9afe0b046700089e48ad38eab607902568c516fcde9aa08354R4)
+
+## Note on v1.1.1 being first version
+Due to mechanical issues with the release machinery's interaction with GOPROXY, the first release of v1.1 is v1.1.1, not v1.1.0
+
+## Changes since v1.0.0
+
+### P2MP
+
+One of the most fundamental components of Network Service Mesh in Kubernetes is the Forwarder.  It handles the mechanics of cross connecting whatever 'local mechanism' (like kernel interface, vfio, or memif) is injected into the Client Pod to whatever local (kernel,vfio, memif) or remote (vxlan, wireguard, etc) mechanism is needed to reach the Network Service Endpoint providing its requested Network Service.
+
+Traditionally, the Forwarder has been Point to Point (P2P). P2P has the benefit of having a single simple contract: a packet(or frame) that goes in one side comes out the other.  
+
+As the number and variety of NSM use cases has grown, some use cases have emerged in which the Forwarder needs to do more complex cross connects than just P2P.  Point to multipoint (P2MP), by contrast to P2P, have a large number of mutually incompatible implementations.  Often P2MP will be spoken of in shorthand as if its is a single unitary thing: it is not.
+
+In order to support these sorts of use cases, NSM made a slight architectual enhancement: have the forwarder select the NSE (or in the case of P2MP NSEs) to which it should be connecting.  This change allows a Forwarder to support zero or more possible P2MP senarios.
+
+Given a Network Service that requires some particular P2MP capabilty (or any other kind of capability) from its Forwarder for one of its logical links, it is necessary to have some capacity of 'hinting' that requirement to the NSMgr so that it selects the correct Forwarder in the event that more than on is present.  NetworkService matches can now be decorated with 'metadata' with 'labels' for use by the NSMgr in selecting the correct Forwarder for each logical hop the match represents.
+
+
+### Interdomain improvements
+
+- [Network Service Mesh Interdomain](http://localhost:1313/docs/concepts/architecture/#inter-domain) now supports Spire Federation.   Spire Federation allows workloads from
+different Trust Domains to connect to a Network Service securely and subject to authorization.
+- [Network Service Mesh Interdomain CI](https://github.com/networkservicemesh/integration-interdomain-k8s) now tests interdomain simultaneously across:
+    - GKE
+    - EKS
+    - AKS
+    - Vanilla K8s on Bare Metal (using Equinix Metal)
+   validating that, for example, a Pod in GKE can connect to a Network Service being provided by a Pod in AKS.
+
```

Comment:
Do we also need to add that we tested release on interdomain ci?
```suggestion
NSM v1.1.1 has been successfully [integration interdomain tested](https://github.com/networkservicemesh/integration-interdomain-k8s/actions/runs/1514861946)
```

Reply:
Does this work: https://github.com/networkservicemesh/site/pull/207/commits/99a0d4b0e59b6e8e723f2ee1a835d5f941610005 ?